Makefile

```diff
@@ -74,3 +74,8 @@ makemessages: assets
 
 compilemessages:
 	python -m kolibri manage compilemessages -- -l en > /dev/null
+
+syncmessages:
```

Comment:
Fancy - is there a reason for not having separate commands for each of these?

`checkcrowdinclient`
`uploadmessages`
`downloadmessages`

then `syncmessages` could just call each of the above in turn?